com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/core/engine/importer/IMigrationImporter.java

```diff
@@ -121,6 +121,13 @@ public interface IMigrationImporter {
 	 * @param view View
 	 */
 	public void createView(View view);
+	
+	/**
+	 * Create view alter
+	 * 
+	 * @param view View
+	 */
+	public void createViewAlter(View view);
```

Comment:
how about change method name to [alterView]. Because looks like the other method's purpose is execute create query, but the purpose of this method is execute an alter query.

Reply:
I added create because I thought is was a method that is responsible for creating the final view alter statement. 
However, it seems more appropriate to see it in the sense of 'executing a view alter query', and this seems more clear.
I'll fix the method name to "alterView".
Thank you